src/main/webapp/js/userMap.js

```diff
@@ -14,20 +14,25 @@ function handleData(err, countryCoordinates, userData) {
         handleError();
         return;
     }
-    var userCountries = Object.keys(userData);
     var countriesArr = [];
     var total = 0;
-    userCountries.forEach(function(countryName) {
-        var countryTotal = userData[countryName].length;
-
-        countriesArr.push([countryName, countryTotal]);
-        total += countryTotal;
-    });
-
+    var countryTotal = 0;
+    var date = userData.lastUpdated;
+    for (key in userData.institutes) {
```

Comment:
I just did a small tinker on my machine and managed to pull it off with just those two lines of change. What is it that cause your unexpected answer?